### PR TITLE
Record video of HIRO training

### PR DESCRIPTION
**Pull request by peasant98**
This PR allows for recording of HIRO evalulation episodes, simply by adding a `--record`.

### CODE REVIEW

File: pfrl/experiments/evaluator.py [hunk 1]
```diff
@@ -82,10 +83,17 @@ def _run_episodes(
 
 def _hrl_run_episodes(
     env, agent: HIROAgent, n_steps, n_episodes, max_episode_len=None, logger=None,
+    step_number=None, video_outdir=None
 ):
     """Run multiple episodes and return returns."""
     assert (n_steps is None) != (n_episodes is None)
 
+
+    if step_number is not None:
```
Comment (watakandai) [inline pfrl/experiments/evaluator.py]: I think you should first assign to a boolean. It's difficult to understand what's happening at first glance

```
record_video = step_number is not None

if record_video: 
...
```

**Review by watakandai — CHANGES_REQUESTED** [review]

File: pfrl/experiments/evaluator.py (hunk 1 above)
Comment (watakandai) [reply pfrl/experiments/evaluator.py]: or we can rename `step_number` to `step_number_for_recording` or something that has `record` component. 
I prefer the former method though.

Comment (watakandai) [reply pfrl/experiments/evaluator.py]: I actually found this feature in `VideoRecorder` called `enabled` variable.

```
        enabled (bool): Whether to actually record video, or just no-op (for convenience)
```

We can turn this on and off whenever `step_number==None`

**Comment by watakandai** [comment]
Once that's fixed, I think this PR is good to be merged.

File: pfrl/experiments/evaluator.py (hunk 1 above)
Comment (peasant98) [reply pfrl/experiments/evaluator.py]: ah great, thx!

Comment (peasant98) [reply pfrl/experiments/evaluator.py]: fixed.

**Review by peasant98 — COMMENTED** [review]
LGTM

File: pfrl/experiments/evaluator.py [hunk 2]
```diff
@@ -108,6 +116,10 @@ def _hrl_run_episodes(
 
         a = agent.act_low_level(obs, sg)
         obs_dict, r, done, info = env.step(a)
+
+        if step_number is not None:
+            video_recorder.capture_frame()
+
```
Comment (watakandai) [inline pfrl/experiments/evaluator.py]: if the video_recorder is not enabled, I think we can also delete this `if`s

**Review by watakandai — CHANGES_REQUESTED** [review]

File: pfrl/experiments/evaluator.py (hunk 2 above)
Comment (peasant98) [reply pfrl/experiments/evaluator.py]: oh good catch.

**Review by watakandai — APPROVED** [review]

**Comment by watakandai** [comment]
LGTM

**Pull request merged**